.github/workflows/sdk.yaml

```diff
@@ -101,9 +102,6 @@ jobs:
       - run: cargo doc --workspace
         env:
           RUSTDOCFLAGS: "-D warnings"
-      - run: cargo doc --package google-cloud-gax
```

Comment:
As I recall, `cargo doc`  can fail to build if you have some features enabled vs. disabled (e.g. if you link an element that is not present because the feature is disabled).   I believe we should keep this test.

Reply:
I reverted all changes to `.github/workflows/sdk.yaml`, but that is not obvious from the 9000 commits associated with this PR.